components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/service/DCRMService.java

```diff
@@ -513,6 +540,25 @@ private Application createOAuthApplication(ApplicationRegistrationRequest regist
         return application;
     }
 
+    /**
+     * Check whether the software statement is mandatory and the software statement is empty.
+     *
+     * @param softwareStatement Software statement.
+     * @return True if the software statement is mandatory and the software statement is empty.
+     * @throws DCRMServerException If an error occurred while checking the software statement.
+     */
+    public boolean isSSAMandatedAndSSAisEmpty(String softwareStatement)
+            throws DCRMServerException {
+
+        DCRConfiguration dcrConfiguration = DCRConfigUtils.getDCRConfiguration();
```

Comment:
Can't we make dcrConfiguration as a class variable.

Reply:
`DCRConfiguration` is a model class and and its instances are used to store data, and the values may not be the same across those instances. Hence we cannot have static class variables.